src/extensions/scratch3_thymio/index.js

```diff
@@ -698,7 +705,11 @@ class Thymio {
         this.sendAction('V_leds_circle', [0, 0, 0, 0, 0, 0, 0, 0], () => {
             this.sendAction('V_leds_top', [0, 0, 0], () => {
                 this.sendAction('V_leds_bottom', [0, 0, 0, 0], () => {
-                    this.sendAction('V_leds_bottom', [1, 0, 0, 0], () => {});
+                    this.sendAction('V_leds_bottom', [1, 0, 0, 0], () => {
+                        if (this.useHorizontalLeds) {
+                            this.sendAction('V_leds_prox_h', [0, 0, 0, 0, 0, 0, 0, 0]);
+                        }
+                    });
                 });
```

Comment:
Could this be batch in a single emitEvents ?

Reply:
Sure, good catch! I'm not at the office for a few days but I will check that when I'm back.